docs/modules/pwm2.md

```diff
@@ -0,0 +1,195 @@
+# PWM Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2019-02-12 | [fikin](https://github.com/fikin) | [fikin](https://github.com/fikin) | [pwm2.c](../../../app/modules/pwm2.c)|
+
+Module to generate PWM impulses to any of the GPIO pins. 
+
+PWM is being generated by software using soft-interrupt TIMER1 FRC1. This timer is used by other modules like ws2812 and pwm, but only one of them can use it at given time. Also some parts of OS might mask the interrupt for awhile.
```

Comment:
Multiple users can use it at the same time....

Reply:
ok, take a look at latest upload and lets see if the situation is improved.